src/test/java/com/securian/creditcompass/OrderCalculatorTest.java

```diff
@@ -0,0 +1,69 @@
+package com.securian.creditcompass;
+
+import com.securian.creditcompass.entities.Claim;
+import com.securian.creditcompass.orderCalculator.OrderCalculator;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class OrderCalculatorTest {
+    private OrderCalculator orderCalculator;
+
+    @Test
+    void init(){
```

Comment:
Use setUp() instead and add to https://github.com/beforeeach decorator before it (signals 'setting up' or 'initialization' that needs to be done before an actual test)